src/kernel-install/90-loaderentry.install

```diff
@@ -63,6 +64,13 @@ if ! [[ ${BOOT_OPTIONS[*]} ]]; then
     exit 1
 fi
 
+if [[ -f /etc/kernel/tries ]]; then
+    read -r TRIES < /etc/kernel/tries
```

Comment:
There's no space after `<` or `>` elsewhere in this file, please remove it here too.